src/app/components/iterations-modal/iterations-modal.component.html

```diff
@@ -1,11 +1,12 @@
 <modal
+  class="f8-iteration-modal"
   #createUpdateIterationDialog
```

Comment:
The modal title is not seen. Also, the active OFF text is not seen.
![createiteration](https://user-images.githubusercontent.com/1892722/30213937-33f42298-949a-11e7-984b-908dff6ea835.png)